tests/src/Functional/ShowcasePageTest.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\node\Entity\Node;
+use Drupal\Tests\oe_showcase\Traits\AuthenticationTrait;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Ensure OE Showcase Pages are created.
```

Comment:
Lets be more descriptive as to what is going on in this test
```suggestion
 * Page content creation.
```